src/utils/fieldUtils/renderOption.js

```diff
@@ -12,13 +12,17 @@ export default function renderOption(opt, idx = 0) {
     reactKey,
     value,
     label,
+    selected,
   } = opt;
+  const selectedOption = selected ? { selected: 'selected' } : {};
+
 
   return (
     <option
       key={reactKey ? `${reactKey}-${idx}` : `opt-${randomStr()}`}
       value={value}
       className={styles['option']}
+      { ...selectedOption }
```

Comment:
I did it this way so it wouldn't put `selected=""` on unselected items. Maybe that's not a big deal though. Thoughts?

Reply:
It seems like it would be undefined not an empty string, and props that are undefined just get stripped by React I believe. Empty strings are falsy anyway.

Reply:
It broke the snapshot when I did it that way. If everything works, I'll see how I can clean this up in the follow up, as well.

Reply:
I don't believe the `undefined` prop will get stripped. I think React will try to place it on the node as `selected=undefined`